### DISCUSSION TITLE
爱桌游 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#136] [模板讨论]
爱桌游 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
2025-02-27 15:02:20
日志：Failed at 2/4 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: {{domain}}lzy/user/sign
